### ISSUE TITLE
DSB stage 2 dataset unzip password wanted 

### ISSUE THREAD

**Issue by lihaossu** [wentaozhu/DeepLung#61]
My model is in test stage, I got kaggle stage 2 dataset without unzip password, can someone share it with me? thank you

**Comment by wentaozhu** [comment]
You may register the kaggle and can get the password.

**Comment by lihaossu** [comment]
> You may register the kaggle and can get the password.

@wentaozhu 
I did but stage2 dataset is impossible to download anymore

**Comment by wentaozhu** [comment]
I think maybe stage 2 is for the final purpose only. Now the competition is finished and there is no access to stage 2.

**Comment by wentaozhu** [comment]
But I will not close the issue so that other people can help you if they can see this issue. Good luck!

**Comment by lihaossu** [comment]
@wentaozhu , Thank you, I already got the stage 2 zip file, so i really want to test my model, just need unzip.txt file, thanks again.

**Comment by aszhanghuali** [comment]
@lihaossu How did you get the stage 2 zip file?I also need it! Could you share it with me ?1319466165@qq.com
Thank you !Looking forward to you reply!

**Comment by MHansy** [comment]
Hello Wentaozhu！
Please help me to understant on these.
1. Can I train this model by CPU?
2. How long it can take to run the process of data preparation?  
This is because I tried to run the prepare.py and the file is still in processing for more than one day.

**Comment by oliva-cool** [comment]
Hello Wentaozhu！
          I am runing the nodcls/data/extclsshpinfo.py,, but I meet the error  No such file or directory: '**/DOI', what the DOI directory  contain?

**Comment by wentaozhu** [comment]
> Hello Wentaozhu！
> I am runing the nodcls/data/extclsshpinfo.py,, but I meet the error No such file or directory: '**/DOI', what the DOI directory contain?

It is LIDC-IDRI dataset https://wiki.cancerimagingarchive.net/display/Public/LIDC-IDRI consisting of dicom images.

**Comment by wentaozhu** [comment]
> Hello Wentaozhu！
> Please help me to understant on these.
> 
> 1. Can I train this model by CPU?
> 2. How long it can take to run the process of data preparation?
>    This is because I tried to run the prepare.py and the file is still in processing for more than one day.

Yes, you can train it by CPU. Data preparation takes a couple of hours. Try to use 32 CPU kernels.

**Comment by oliva-cool** [comment]
> > Hello Wentaozhu！
> > I am runing the nodcls/data/extclsshpinfo.py,, but I meet the error No such file or directory: '**/DOI', what the DOI directory contain?
> 
> It is LIDC-IDRI dataset https://wiki.cancerimagingarchive.net/display/Public/LIDC-IDRI consisting of dicom images.

Thanks , can I replace it with luna16 datasets? It is too slow to download  the LIDC-IDRI dataset in a server .

**Comment by wentaozhu** [comment]
Yes. You can. But LUNA16 has no classification label.

**Comment by oliva-cool** [comment]
> Yes. You can. But LUNA16 has no classification label.

I see, thank you very much!

**Comment by MHansy** [comment]
> 
> 
> > Hello Wentaozhu！
> > Please help me to understant on these.
> > 
> > 1. Can I train this model by CPU?
> > 2. How long it can take to run the process of data preparation?
> >    This is because I tried to run the prepare.py and the file is still in processing for more than one day.
> 
> Yes, you can train it by CPU. Data preparation takes a couple of hours. Try to use 32 CPU kernels.

Thanks! I did it.

**Comment by MHansy** [comment]
![image](https://user-images.githubusercontent.com/64502395/116550162-fdfe0080-a928-11eb-9397-e3892aedc1b3.png)

Hello Wentouzhu!

Kindly help me to know, what does tpr, tnr, total pos, and total neg mean??

**Comment by wentaozhu** [comment]
> ![image](https://user-images.githubusercontent.com/64502395/116550162-fdfe0080-a928-11eb-9397-e3892aedc1b3.png)
> 
> Hello Wentouzhu!
> 
> Kindly help me to know, what does tpr, tnr, total pos, and total neg mean??

Please create a new issue. It does not belong to this issue. Please check the paper for the detail Zhu, Wentao, Chaochun Liu, Wei Fan, and Xiaohui Xie. "DeepLung: Deep 3D Dual Path Nets for Automated Pulmonary Nodule Detection and Classification." IEEE WACV, 2018. Thank you!

It is the standard TPR, TNR in ROC.
